vip-cache-manager.php

```diff
@@ -8,5 +8,5 @@
 License: GPL version 2 or later - http://www.gnu.org/licenses/old-licenses/gpl-2.0.html
 */
 
-require_once __DIR__ . '/vip-cache-manager/vip-cache-manager.php';
+require_once __DIR__ . '/vip-cache-manager/class-wpcom-vip-cache-manager.php';
```

Comment:
> Class file names should be based on the class name with "class-" prepended. Expected class-wpcom-vip-cache-manager.php, but found vip-cache-manager.php. (WordPress.Files.FileName.InvalidClassFileName)

Reply:
I'm not sure if this is a standard we wish to adhere to.

Reply:
No problem, should we add an exclusion to phpcs.xml?

Reply:
> No problem, should we add an exclusion to phpcs.xml?

Yes, please

Reply:
Fixed in 8d5b9a8e2